spec/features/routing/full_scenario_spec.rb

```diff
@@ -22,11 +25,17 @@
 
     # rename defaut groupe to littéraire
     click_on 'voir'
-    fill_in 'groupe_instructeur_label', with: 'littéraire'
+    wait_until { page.has_css?('#groupe_instructeur_label') }
```

Comment:
Normalement c'est équivalent à `expect(page).to have_css('#groupe_instructeur_label')`  (`have_selector` fait partie des matchers qui cherchent le contenu tant que le `wait_time` par défaut n'a pas expiré).

https://thoughtbot.com/blog/write-reliable-asynchronous-integration-tests-with-capybara#summary

Reply:
Trop cool, merci pour le complément ! Mes recherches donnaient des résultats assez naïfs, content de voir un meilleur article qui clarifie le comportement

Reply:
Yes, j'ai remplacé tous les wait_until par des expect